client/blocks/comment-button/style.scss

```diff
@@ -1,26 +1,29 @@
 .comment-button {
 	align-items: center;
 	box-sizing: border-box;
-	color: var( --color-text-subtle );
 	cursor: pointer;
 	display: inline-flex;
 	list-style-type: none;
 	padding: 4px;
 	position: relative;
 
-	&:hover,
-	&:active {
-		color: var( --color-primary );
+	.comment-button__label-status {
+		@include breakpoint-deprecated( '<480px' ) {
```

Comment:
Is there a non-deprecated one we should use instead?

Reply:
The entire Reader layout is based on the old breakpoints, so for the moment I'm reluctant to dive into that particular rabbit hole 🐰